tests/parse/test_parse_case.py

```diff
@@ -229,6 +229,20 @@ def test_parse_case_multiple_alignment_files(scout_config):
         assert ind["bam_file"] == cram_path
 
 
+def test_parse_case_RNA_alignment_files(scout_config):
```

Comment:
```suggestion
def test_parse_case_rna_alignment_files(scout_config):
```

Reply:
I think it's a PEP thing to have all functions with names in lowercase -> https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
I call on the previous standard - also the established arcronym standard. I guess I could buy `test_parse_case_ribonucleic_acid_alignment_files()`..